pkg/api/types.go

```diff
@@ -921,6 +921,9 @@ type PodSpec struct {
 	// used must be specified.
 	// Optional: Default to false.
 	HostNetwork bool `json:"hostNetwork,omitempty"`
+	// Run the container using the host ipc mode.
+	// Optional: Default to false.
+	HostIPC bool `json:"hostIPC,omitempty"`
```

Comment:
ah, misread... we're currently using the infra pod's id as the IPC namespace, so it is shared. should `createPodInfraContainer` create the infra pod in the host IPC namespace also if `pod.spec.HostIPC` is true?